utils/ctranslate.py

```diff
@@ -26,17 +27,25 @@ def __init__(self, model, tokenizer, compute_type):
         self.__times_invoked = 0
         self.__start_times = list()
         self.__finish_times = list()
+        self.__is_profiling = aio_profiler_enabled()
+        self.__first_run = True
```

Comment:
I know that I was the first to wrongly use double underscores to make attributes of python classes "private" but no need to keep on repeating my mistakes! 😃